libexec/revng/mass-testing/revng-revng-test

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+set -euo pipefail
+
+# This is a revng-specific script that simplifies using the `revng` together
+# with test-harness. In particular it does the following:
+# * Saves `sections.json` and adds `text_size` to meta.yml
+# * Saves the trace output as `trace.json.gz`
+# * Suppresses output from the `revng` command
+# * Detects if it's being used as a wrapper for `revng` and calls the real one
+#   properly
+
+SECTIONS_JSON="$TEST_OUTPUT_DIR/sections.json"
+
+dump-sections "$TEST_INPUT" > "$SECTIONS_JSON"
```

Comment:
> How is dump-sections in PATH? o_O

`revng mass-testing run` adds the `mass-testing` executables to `PATH` without the prefix [Link](https://github.com/revng/revng/pull/396/commits/a533af74d1fc19b5026f88f1d3f57c3f73b84ef7#diff-9b1da3fad311f65152340658285d7a8adf1a9f03e79fa5949720db0e87c364f7R44-R55)

Reply:
Broken link

Reply:
Here's the new one: https://github.com/revng/revng/pull/396/commits/b6c636b81789ae016835d3d9ea5784dffe996972#diff-9b1da3fad311f65152340658285d7a8adf1a9f03e79fa5949720db0e87c364f7R44-R52

Reply:
I'm confused, what's the ratio for creating symlinks without `revng-`?

Reply:
So that you can write:
```
test-harness ... ---
  revng ...
```
in `commands.yml` without `revng mass-testing`

Reply:
With:
```yaml
commands:
- type: revng-regression.decompile
  from:
  - type: source
  suffix: /
  command: |-
    test-harness "$INPUT" "$OUTPUT" --timeout 300 --
    revng
      artifact
      decompile-to-single-file
      --analyze %INPUT%
```

Without:
```yaml
commands:
- type: revng-regression.decompile
  from:
  - type: source
  suffix: /
  command: |-
    revng mass-testing test-harness "$INPUT" "$OUTPUT" --timeout 300 --
    revng mass-testing revng-test
      artifact
      decompile-to-single-file
      --analyze %INPUT%
```

(I've also changed the `revng` invocation because I have no idea how `revng` would be symlinked to `revng-test`, since it would no longer be on `PATH`, modulo implementing some lookup in `libexec`)